react-client/src/views/admin/EditStudent.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from 'react';
+import Button from 'react-bootstrap/Button';
+import { useNavigate, useParams } from 'react-router-dom';
+import {
+  CCard,
+  CCardBody,
+  CCardHeader,
+  CCol,
+  CRow,
+  CForm,
+  CFormLabel,
+  CFormInput
+} from '@coreui/react'
+import { toast } from 'react-toastify';
+import { useQuery, useMutation } from '@apollo/client';
+import { UPDATE_STUDENT } from '../../graphql/mutations';
+import { GET_STUDENT } from '../../graphql/queries';
+
+function UpdateStudent(props) {
+  const [student, setStudent] = useState([]);
+  const params = useParams();
+  const id = params.studentId;
+  //console.log('in Update Student params= ',params);
+  console.log('in Update Student id= ', id);
+  const { data, loading, error } = useQuery(GET_STUDENT, {
+    variables: { id },
+    onCompleted: (data) => {
+      const { student } = data;
+      console.log('student= ',student);
+      setStudent(student);
+    },
+  });
+
+  if (error) { console.log('error= ',error);}
+  if (data) { console.log('data= ',data);}
+
+  const [updateStudentMutation] = useMutation(UPDATE_STUDENT);
+
+  const updateStudent = async (event) => {
+    event.preventDefault();
+    console.log('in handleSubmit, user=', student);
+    
+    try {
+      console.log('Before updateUser call');
+      console.log('student= ',student);
+      console.log('id= ',id);
+      await updateStudentMutation({
+        variables: { id, ...student },
+      });
+      console.log('After updateUser call', student);
+      navigate('/userlist');
```

Comment:
@dagutierrez-mx here needs to be `navigate('/admin/students')`

Reply:
Done